### PR TITLE
✨  Add Spotify Login Button

### LINKED ISSUE #7: Login Page

### PR DESCRIPTION
**Pull request by christophvogel**
Added Spotify Login Button with Logo

### CODE REVIEW

File: components/Login/LoginButton.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.login {
+  background-color: var(--spotify-color);
+
+  /* padding: 1em; */
```
Comment (fre-ben) [inline components/Login/LoginButton.module.css]: Commentary could be removed

File: components/Login/LoginButton.module.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  border-radius: 50px;
+  border: none;
+  font-family: var(--primary-font);
+  font-size: 16pt;
```
Comment (fre-ben) [inline components/Login/LoginButton.module.css]: Use either px or pt - should be the same I think?

File: components/Login/LoginButton.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styles from "./LoginButton.module.css";
+
+function LoginButton() {
+  function handleClick(e) {
+    e.preventDefault();
+    console.log("You will be redirected to spotify.");
+  }
+
+  return (
+    <button className={`${styles.login}`}>
```
Comment (fre-ben) [inline components/Login/LoginButton.tsx]: You could also write `<button className={styles.login}>`

File: components/Login/LoginButton.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      <img src="/spotify-icon.svg" className={`${styles.spotify}`} />
```
Comment (fre-ben) [inline components/Login/LoginButton.tsx]: Same here, there's no need for the backticks inside the className.

File: components/Login/LoginButton.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      <a href="#" onClick={handleClick}>
```
Comment (fre-ben) [inline components/Login/LoginButton.tsx]: Why use an `a` tag?

**Review by fre-ben — CHANGES_REQUESTED** [review]

**Review by fre-ben — COMMENTED** [review]
Your checks seem to fail, because you haven't prettified all your files.
Check out the error and run prettier ;)

File: components/Login/LoginButton.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        Login with Spotify
+      </a>
+    </button>
```
Comment (m-wiemers) [inline components/Login/LoginButton.tsx]: Is it better to place the `<a>` around the button? So the user can click anywhere at the button and not just at the text...

```HTML
<a>
   <button>
     <img>
   </button>
</a>
```

**Review by m-wiemers — APPROVED** [review]
I'm sure I'll sleep well tonight 💤

File: components/Login/LoginButton.tsx (hunk 5 above)
Comment (christophvogel) [reply components/Login/LoginButton.tsx]: There will be a link to spotify authentication later.

File: components/Login/LoginButton.tsx (hunk 6 above)
Comment (christophvogel) [reply components/Login/LoginButton.tsx]: Yes, i will change that!

File: components/Login/LoginButton.module.css (hunk 2 above)
Comment (lmachens) [reply components/Login/LoginButton.module.css]: Another option is to use relative sizes like `em` or `rem`.

File: components/Login/LoginButton.module.css [hunk 7]
```diff
@@ -0,0 +1,15 @@
+.login {
+  background-color: var(--spotify-color);
+  border-radius: 50px;
+  border: none;
+  font-family: var(--primary-font);
+  font-size: 16px;
+  display: flex;
+  align-items: center;
+  width: 315px;
+  height: 52px;
```
Comment (lmachens) [inline components/Login/LoginButton.module.css:10]: Usually, you don't need a fix `width` and `height`.
Instead, give the `img` inside your `button`  a `height` relative to your `font-size`.
Now, you can adjust the `padding` of your `button`.

**Review by lmachens — COMMENTED** [review]
Gj!

**Pull request merged**